network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -278,151 +280,136 @@ public static String buildCloneNetworksQuery(Collection<String> columns) {
                 "where uuid = ? and " + VARIANT_NUM_COLUMN + " = ?";
     }
 
-    // Temporary Limits
-    public static String buildCloneTemporaryLimitsQuery() {
-        return "insert into temporarylimit(" + EQUIPMENT_ID_COLUMN + ", " + EQUIPMENT_TYPE_COLUMN + ", " +
-                NETWORK_UUID_COLUMN + ", " + VARIANT_NUM_COLUMN + ", " + OPERATIONAL_LIMITS_GROUP_ID_COLUMN + ", " + SIDE_COLUMN + ", " + LIMIT_TYPE_COLUMN + ", " + NAME_COLUMN +
-                ", value_, acceptableDuration, fictitious) " + "select " + EQUIPMENT_ID_COLUMN + ", " +
-                EQUIPMENT_TYPE_COLUMN + ", ?, ?, " + OPERATIONAL_LIMITS_GROUP_ID_COLUMN + ", " + SIDE_COLUMN + ", " + LIMIT_TYPE_COLUMN + ", " + NAME_COLUMN +
-                ", value_, acceptableDuration, fictitious from temporarylimit where " + NETWORK_UUID_COLUMN +
-                " = ? and " + VARIANT_NUM_COLUMN + " = ?";
+    // temporary limits
+    public static String buildInsertNewTemporaryLimitsQuery() {
+        return "insert into " + TEMPORARY_LIMITS + "(" +
+            EQUIPMENT_ID_COLUMN + ", " + EQUIPMENT_TYPE_COLUMN + ", " +
+            NETWORK_UUID_COLUMN + ", " +
+            VARIANT_NUM_COLUMN + ", " + TEMPORARY_LIMITS + ") " +
+            " values (?, ?, ?, ?, ?)";
     }
 
-    public static String buildTemporaryLimitQuery(String columnNameForWhereClause) {
```

Comment:
In all this file, try not to move the code and just change the name or the content of the methods